src/monitors/images.ts

```diff
@@ -35,40 +36,54 @@ botCache.monitors.set("images", {
       }] Processing.`,
     );
 
-    message.attachments.forEach(async (attachment) => {
+    const embed = botCache.helpers.authorEmbed(message)
+      .setDescription([
+        translate(message.guildID, "strings:CHANNEL", {
+          channel: `<#${message.channelID}>`,
+        }),
+        translate(message.guildID, "strings:MESSAGE_ID", {
+          id: `<#${message.id}>`,
+        }),
+        translate(message.guildID, "strings:USER", {
+          tag: message.member?.tag || message.author.username,
+          id: `<#${message.author.id}>`,
+        }),
+      ]);
+
+    for (const attachment of message.attachments) {
       const blob = await fetch(attachment.url).then((res) => res.blob()).catch(
         console.log,
       );
       if (blob) {
         await sendEmbed(
           logs.imageChannelID,
-          botCache.helpers.authorEmbed(message).attachFile(
+          embed.attachFile(
             blob,
             attachment.filename,
           ),
         )?.catch(console.log);
```

Comment:
```suggestion
        )
```
(sendEmbed() catches it self now)